app/cdap/components/hydrator/components/TopPanel/ActionButtons/styles.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import styled from 'styled-components';
+import { Button, Tooltip, withStyles } from '@material-ui/core';
+import IconSVG from 'components/shared/IconSVG';
+
+export const ActionButtonsContainer = styled.div`
+  display: inline-block;
+  min-width: 460px;
+  position: relative;
+  text-align: initial;
+  vertical-align: middle;
+`;
+
+export const BaseButton = styled(Button)`
```

Comment:
Can you base this on `PrimaryTextButton`? It looks like all the buttons in the component use the text variant.

Reply:
Actually, on second thought, maybe not? If a component is heavily customized, maybe we shouldn't use the wrapped component from `shared`. Thoughts?

Reply:
I don't have strong preference, here indeed we could base on `PrimaryTextButton` since all the buttons are text button